tester_app/app/assets/javascripts/application.js

```diff
@@ -10,7 +24,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
-//= require jquery_ujs
+
+
```

Comment:
single blank line

take a look at my recent change in the tutorial for this file.